Sources/ReaderDocument.m

```diff
@@ -257,21 +260,6 @@ - (NSURL *)fileURL
 	return _fileURL;
 }
 
-- (BOOL)canEmail
-{
-	return YES;
-}
-
-- (BOOL)canExport
-{
-	return YES;
-}
-
-- (BOOL)canPrint
-{
-	return YES;
-}
-
```

Comment:
Awesome!!! Keep rocking!! Let me know if I can assist with code review or something like that :)